src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationHandlers.Discovery.cs

```diff
@@ -258,6 +258,15 @@ public ValueTask HandleAsync(HandleConfigurationResponseContext context)
                     context.Configuration.ScopesSupported.Add(Scopes.OpenId);
                 }
 
+                // Genesys supports oAuth but doesn't format the "openid" scope using the standard casing.
+                // To ensure Genesys is not treated as an OAuth 2.0-only provider, the invalid "OpenId" scope is
+                // removed from the list and the "openid" value is added to indicate OpenID Connect is supported.
+                else if (context.Registration.ProviderType is ProviderTypes.Genesys)
```

Comment:
Why do you need that? The provider uses a static configuration, so a configuration request will never be sent. It's basically dead code 😃

Reply:
Please revert all the changes made to that file (formatting and logic changes).